sqlitecluster/SQLiteNode.cpp

```diff
@@ -1668,13 +1796,6 @@ void SQLiteNode::_onDisconnect(Peer* peer) {
         // If there were escalated commands, give them back to the server to retry, unless it looks like they were in
         // progress when the leader died, in which case we say they completed with a 500 Error.
         for (auto& cmd : _escalatedCommandMap) {
-            // If this isn't set, the leader hadn't actually started processing this, and we can re-queue it.
```

Comment:
It's possible we've seen it at some point. It doesn't seem like we've seen it recently. It can work at best, sometimes, and it's likely not to work at all.